classes/Mail.php

```diff
@@ -146,6 +146,12 @@ public static function Send($id_lang, $template, $subject, $template_vars, $to,
             $from_name = null;
         }
 
+		// If $to is a string list of e-mail addresses separated by ';' => conversion to array()
+		$is_to_string_list = strpos($to, ';');
```

Comment:
Hi,

thank you fo ryour PR. You have an indentation issue, tabs instaed of spaces probably.

Regards